interpreter/valid/valid.ml

```diff
@@ -416,6 +420,8 @@ let check_global_type (gt : global_type) at =
   let GlobalType (t, mut) = gt in
   check_value_type t at
 
+let check_exn_type (ft : exn_type) at =
+  check_func_type ft at
```

Comment:
This isn't enough, you also need to check that the result is empty.

Reply:
I have updated this now.